src/scenarios.jl

```diff
@@ -1,64 +1,91 @@
+"""
+Construct Scenarios from a SortedDict of Arrays.
+
+# Arguments
+- `name::AbstractString`: user-defined name
+- `input_data::AbstractDict{Dates.DateTime, Matrix{Float64}}`: time series data.
```

Comment:
The function signature uses Array instead of Matrix.  These need to be consistent.